### DISCUSSION TITLE
Adding LazyDocker to lunarVim

### DISCUSSION THREAD

**Discussion by MaymoonaAlBoloshi** [LunarVim/LunarVim#4177] [Q&A]
I'm not too bright on confusing lunarVim as for my liking. 
I was really happy that when I installed Lazygit `<space>gg` worked out of the box 
Is there a way I can add LazyDocker to a key binding, I did install it, it works if I run it through the command line
I am just not sure how to run it like lazygit :/

**Comment by tmbernardo** [comment]
Old thread, but I figured it out for anybody that could find this useful.

Here's how it fits in my config (binded to `<leader>DD`):
```Lua
lvim.builtin.which_key.mappings.D = {
  name = "Docker",
  D = {
    function()
      local Terminal = require("toggleterm.terminal").Terminal
      local lazydocker = Terminal:new {
        cmd = "lazydocker",
        hidden = true,
        direction = "float",
        float_opts = {
          border = "none",
          width = 100000,
          height = 100000,
          zindex = 200,
        },
        on_open = function(_)
          vim.cmd "startinsert!"
        end,
        on_close = function(_) end,
        count = 99,
      }
      lazydocker:toggle()
    end,
    "Lazydocker"
  }
}
```

stolen from their lazygit toggleterm implementation here:
https://github.com/LunarVim/LunarVim/blob/85ccca97acfea9a465e354e18bb2f6109ba417f8/lua/lvim/core/terminal.lua#L153-L172
